web/content/js/versions.js

```diff
@@ -1,695 +1,683 @@
+            "path": "Release_notes"
           },
           {
-            title: "Planning Guide",
-            path: "Planning_Guide"
+            "title": "Planning Guide",
+            "path": "Planning_Guide"
           },
           {
-            title: "Quickstart Guide",
-            path: "Quickstart_Guide"
+            "title": "Quickstart Guide",
+            "path": "Quickstart_Guide"
           },
           {
-            title: "Installing Katello Server",
-            path: "Installing_Server_on_Red_Hat"
+            "title": "Installing Katello Server",
+            "path": "Installing_Server_on_Red_Hat"
           },
           {
-            title: "Installing Smart Proxy with Content",
-            path: "Installing_Proxy_on_Red_Hat"
+            "title": "Installing Smart Proxy with Content",
+            "path": "Installing_Proxy_on_Red_Hat"
           },
           {
-            title: "Upgrading and Updating",
-            path: "Upgrading_and_Updating"
+            "title": "Upgrading and Updating",
+            "path": "Upgrading_and_Updating"
           },
           {
-            title: "Configuring Smart Proxies with a Load Balancer",
-            path: "Configuring_Load_Balancer"
+            "title": "Configuring Smart Proxies with a Load Balancer",
+            "path": "Configuring_Load_Balancer"
           },
           {
-            title: "Content Management Guide",
-            path: "Content_Management_Guide"
+            "title": "Content Management Guide",
+            "path": "Content_Management_Guide"
           },
           {
-            title: "Converting a Host to RHEL",
-            path: "Converting_a_Host_to_RHEL"
+            "title": "Deploying Foreman on AWS",
+            "path": "Deploying_on_AWS"
           },
           {
-            title: "Deploying Foreman on AWS",
-            path: "Deploying_on_AWS"
+            "title": "Provisioning Guide",
+            "path": "Provisioning_Guide"
           },
           {
-            title: "Provisioning Guide",
-            path: "Provisioning_Guide"
+            "title": "Configuring Foreman to use Ansible",
+            "path": "Configuring_Ansible"
           },
           {
-            title: "Configuring Foreman to use Ansible",
-            path: "Configuring_Ansible"
+            "title": "Managing Hosts Guide",
+            "path": "Managing_Hosts"
           },
           {
-            title: "Managing Hosts Guide",
-            path: "Managing_Hosts"
+            "title": "Administering Foreman",
+            "path": "Administering_Red_Hat_Satellite"
           },
           {
-            title: "Administering Foreman",
-            path: "Administering_Red_Hat_Satellite"
-          },
-          {
-            title: "Application Centric Deployment",
-            path: "Application_Centric_Deployment"
+            "title": "Application Centric Deployment",
+            "path": "Application_Centric_Deployment"
```

Comment:
Thanks Github for showing all theses changes from line 531 onwards .... 💯

Reply:
It's in a separate commit, the last commit was mostly `%s/\([a-z]\+\):/"\1":/` in vim.

Reply:
So you can view https://github.com/theforeman/foreman-documentation/pull/1337/commits/1029d7fe5157399c067a806a4abf568a339bd77e right now. Link will break if I push a new commit.